JoinColony/colonyCDapp
# Fix: Main Content and Page Header constraints adjustments

## Description

![image](https://github.com/user-attachments/assets/8d601ac6-5103-4fb6-a386-e94a9f136acf)

## Testing

1. Set the browser width to 1440px
2. Verify that main content is 1144px in width
3. Verify that the distance between the Colony Sidebar and the main content is 32px
4. Verify that the distance between the header and the main content is 32px
5. Verify that the User Pills are centred inside their container when the Action Form is up

Resolves #3094
